src/Avalonia.Controls/Primitives/Popup.cs

```diff
@@ -256,8 +256,8 @@ public PlacementMode PlacementMode
         /// </remarks>
         public Rect? PlacementRect
```

Comment:
`PlacementRect` seems like an unnecessary deviation from naming conventions and WPF which uses `PlacementRectangle`. Honestly, I would like to switch to WPF terminology: https://learn.microsoft.com/en-us/dotnet/api/system.windows.controls.primitives.popup.placementrectangle?view=windowsdesktop-7.0

Reply:
I guess there is precedent for the new naming: https://learn.microsoft.com/en-us/uwp/api/windows.ui.input.pointerpointproperties.contactrect?view=winrt-22621

`PointerPointProperties.ContactRect`